release-notes/opensearch-release-notes-1.3.0.md

```diff
@@ -233,46 +233,46 @@ OpenSearch Dashboards [Release Notes](https://github.com/opensearch-project/Open
 * Changes the default policy of the visual editor to be an empty one ([#149](https://github.com/opensearch-project/index-management-dashboards-plugin/pull/149))
 
 
-### OpenSearch Knn
+### OpenSearch K-NN
```

Comment:
It's small k. k-NN :)
Ref: https://github.com/opensearch-project/k-NN

Reply:
All of our plugin repos are using lower case for the first letter. (e.g. alerting, security) But here we capitalize the first letters of them. (e.g. OpenSearch Alerting, OpenSearch Security). I believe we should follow the same convention for `k-NN` in this case. 
Or in another case, I know k-NN is a specific statistics term. We should use its scientific term then. WDYT?

Reply:
I just saw from previous notes. We used `k-NN` all the time. I will changed the name to be the same with them. :D

Reply:
I would suggest to stay with `k-NN` following https://github.com/opensearch-project/opensearch-build/blob/main/release-notes/opensearch-release-notes-1.2.0.md#opensearch-k-nn

Reply:
Changed. Thanks!